app/src/main/java/com/simple/checkstyledemo/MainActivity.java

```diff
@@ -1,13 +1,23 @@
 package com.simple.checkstyledemo;
 
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.app.Application;
 
 public class MainActivity extends AppCompatActivity {
 
+    private int s;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        int i = 3;
+        test(i);
+    }
+
+    private void test(int a) {
+        if (a > 3) return;
+        System.out.println(a);
```

Comment:
**Reporter**: PMD
**Rule**: SystemPrintln
**Severity**: ERROR
**File**: app/src/main/java/com/simple/checkstyledemo/MainActivity.java L21

Java Logging https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/logging-java.html#SystemPrintln

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*